submissions/reactjs/assignment3.MD

```diff
@@ -7,7 +7,7 @@
 | Pranjal Pandey                 | 2203031080144        |                   |             |
 | Ashish Yadav                   | 2203031080161        |                   |             |
 | Vinit Patel                    | 2203031080139        |                   |             |
-| Utsav Hirapra                  | 2203031080002        |                   |             |
+| Utsav Hirapra                  | 2203031080002        |  [Assignment-33](https://github.com/utsav1213/ReactAssignments/blob/main/src/App.jsx)  | [GitHub](https://github.com/utsav1213/ReactAssignments/)            |
```

Comment:
Please follow the repository structure mentioned in the problem statement document